### ISSUE TITLE
problems running Sagefy

### ISSUE THREAD

**Issue by Iftahh** [heiskr/sagefy#306]
I tried running Sagefy and hit several problems -  I tried following setup.md in docs.

1. Mounting  /vagrant failed -  turns out my VirtualBox version was incompatible with the version in the Vagrant VM - I had to update my local VirtualBox (I had version 5.0 and needed 5.1) 

2. `python3 dev_data.py` failed because it had no config.py.   I copied config.dev.example.py into config.py

3. webpack failed because `ERROR in Cannot find module 'babel-core'`.  I installed babel-core  (ran `npm install bable-core` from the '/vagrant/client' dir)

4. Next I hit a problem that I'm currently stuck in - while trying to log in (or sign up) the server returns "Internal Server Error"  (just string, not JSON).  I exploring how to debug it further, there is nothing related to the error in the `pm2 logs` output

**Comment by heiskr** [comment]
Hi @Iftahh 

Thanks for taking brave road of trying to run it locally :) The process is definitely still pretty messy. I'll look over these notes and get back.

Kevin

**Comment by heiskr** [comment]
Hi @Iftahh 

I've updated the `setup.md` file to include the step to copy `config.py` file (amazing catch!) and `babel-core` to the `package.json`.

The web service is in Python, what's running in `pm2` (node) is two things:
1. a Node server for "isomorphic" rendering
2. the client-side build tools (CSS, JS, etc)

You might want to try restarting the web service with (within the Vagrant box):
```
sudo uwsgi --stop /tmp/uwsgi-master.pid
sudo uwsgi --ini /var/www/setup/uwsgi_local.ini
```

Let me know if that works. It's more fickle than I'd like it to be.

**Comment by Iftahh** [comment]
I can confirm that everything is working now - I'm not sure what was wrong the first time I tried.

I had to restart the Vagrant VM  that I left running earlier, as it became unresponsive (100% CPU, ssh stuck). This might have been because my hard-disk became 100% full and all hell broke loose.

Now after the VM restart I ran uwsgi, nginx, npm  as described in setup.md and everything just worked fine.

Excellent !
